src/status_im2/common/animated_header_list/style.cljs

```diff
@@ -0,0 +1,71 @@
+(ns status-im2.common.animated-header-list.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.reanimated :as reanimated]))
+
+(defn container-view
+  [view-height]
+  {:position :absolute
+   :top      0
+   :left     0
+   :right    0
+   ;; height must be set, otherwise list will not scroll
+   :height   view-height})
+
+(defn button-container
+  [position]
+  (merge
+   {:width            32
+    :height           32
+    :border-radius    10
+    :justify-content  :center
+    :align-items      :center
+    :background-color (colors/theme-colors colors/white-opa-40 colors/neutral-80-opa-40)
+    :position         :absolute
+    :top              56
+    :z-index          3}
+   position))
+
+(defn blur-view
+  [animation]
+  (reanimated/apply-animations-to-style
```

Comment:
I agree with this. I started writing animation styles this way as suggested by Brian. What are your thought @briansztamfater?

Reply:
cc'ing @flexsurfer as well.

Reply:
@ilmotta sometimes when a component has multiple animations defined, it can start cluttering the view component. I can see your point, but maybe defining them in style.cljs would be better, especially since this hook is not affecting the view or causing re-renders, its only mutating the style, so why keep it in the view

Reply:
It's a matter of responsibilities and consistency. The guidelines suggest the style namespaces exist to declutter the view from defining style maps. If we start to move hooks to the `style` namespace we enter a nebulous territory where stateful hooks can be called anywhere. I don't know if the analogy will help, but in React land, CSS modules have a very narrow purpose, and that's why they serve their purpose so well.

>@ilmotta sometimes when a component has multiple animations defined, it can start cluttering the view component

So then the view can create a separate function to do all the animation calls, for example.

-----

At the very least, I think we should either change the guidelines or respect it, otherwise they get outdated and become pointless.

Reply:
Please, take my previous comments with a grain of salt ;) You can close this thread if you want @OmarBasem 

>I experimented with refactoring based on my own suggestion in comment https://github.com/status-im/status-mobile/pull/14925/files#r1091779515, but I didn't like what I saw :man_shrugging: So it's all good from my end.
>
>-- https://github.com/status-im/status-mobile/pull/14925#pullrequestreview-1277033921

Reply:
Just opened a PR https://github.com/status-im/status-mobile/pull/14941 after reviewing all `style.cljs` files in the codebase, and motivated by this thread.

Thanks @OmarBasem for the discussion.

Reply:
My please @ilmotta